packages/create-wdio/src/cli/utils.ts

```diff
@@ -93,27 +115,19 @@ export const parseAnswers = async function (yes: boolean): Promise<ParsedAnswers
     /**
      * find relative paths between tests and pages
      */
-    const hasRootTSConfig = await fs.access(path.resolve(projectRootDir, 'tsconfig.json')).then(() => true, () => false)
-    const tsConfigFilePath = !hasRootTSConfig
+    const hasRootTSWdioConfig = await fs.access(path.resolve(projectRootDir, 'tsconfig.wdio.json')).then(() => true, () => false)
```

Comment:
Ah, I need to edit this to use the constant.
```suggestion
    const hasRootTSWdioConfig = await fs.access(path.resolve(projectRootDir, TSCONFIG_FILE_NAME)).then(() => true, () => false)
```